go/vt/sqlparser/sql.y

```diff
@@ -525,11 +530,11 @@ func yySpecialCommentMode(yylex interface{}) bool {
 %type <queryOpts> query_opts
 %type <str> key_type key_type_opt
 %type <str> flush_type flush_type_opt
-%type <empty> to_opt to_or_as as_opt column_opt
+%type <empty> to_or_as_opt to_or_as /*to_opt*/ as_opt column_opt
```

Comment:
It definitely can, I left it in case anyone needed this rule in the future. It's commented out because the compiler complains if a rule isn't being used anywhere.